src/utils/send-it.js

```diff
@@ -1,8 +1,28 @@
 const fs = require('fs');
+const inquirer = require('inquirer'); // ESLint mad. code still runs
```

Comment:
Ah I see. The linter believes that `inquirer` is required for the lambda to run because we currently have it in the `src` folder (`src/utils/send-it.js`). It is assumed that all code in `src` is required for that lambda, and we accidentally broke that assumption.

Since this is a utility script used only locally by us, let's move it under the `test` folder (`test/utils/send-it.js`)!

Reply:
We can interpret `src` as "minimum code lambda needs to run". And `test` as "all other code e.g. unit tests and utility scripts.

Reply:
Side note: other nomenclature uses `main` instead of `src`